### PR TITLE
Complete user stories for shelter reviews

### PR DESCRIPTION
**Pull request by robertorodriguez12**
Author: Roberto
Contributions: Priya & Roberto
- Users will be able to share their experiences with shelter by providing reviews. Users should be able to create a review with a title (Example: "Awesome place!"), a rating (out of 5), and content (Example: "Truly enjoyed our time working with this shelter. Staff was great, and we found our perfect pet!"). A user can also upload one picture (image URL address) for their review as well, but this is optional.
- Improvements: We need a limiter on our ratings and possibly the number of characters in content
- Edge Cases: What happens when a user doesn't upload a URL? 
- View: Increase the size of the box for content 
    - Explore migrating consistent HTML to application.html.erb and consistent CSS to CSS style sheets; do research into what pages look like after (W3)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,6 +26,4 @@
 # Ignore master key for decrypting credentials and more.
 /config/master.key
 
-# Ignore test-related files
-/coverage
-/coverage/
+coverage
```
Comment (priyapower) [inline .gitignore:29]: Awesome job! We finally got this working

Comment (robertorodriguez12) [reply .gitignore:29]: Thank you!

File: app/models/review.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Review < ApplicationRecord
+  belongs_to :shelter
+
+  validates_presence_of :title
+  validates_presence_of :rating
+  validates_presence_of :content
+  validates_presence_of :image, :allow_blank => true
```
Comment (priyapower) [inline app/models/review.rb:7]: Can you explain how the allow_blank works?

File: app/models/shelter.rb [hunk 3]
```diff
@@ -1,5 +1,6 @@
 class Shelter < ApplicationRecord
   has_many :pets, dependent: :destroy
+  has_many :reviews, dependent: :destroy
```
Comment (priyapower) [inline app/models/shelter.rb:3]: Awesome! I see our new reviews association for shelter

**Comment by priyapower** [comment]
Reviewed by: Priya
Looks aweseome!
The only things I see moving forward is refactoring and updating the views :)

File: app/models/review.rb (hunk 2 above)
Comment (robertorodriguez12) [reply app/models/review.rb:7]: It allows for no image to be added.

**Pull request merged**